test/built-ins/Temporal/Calendar/prototype/prop-desc.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
```

Comment:
Could you update this to 2021? (Sorry I missed this the first time around)

Reply:
Yep, done!